routes/index.js

```diff
@@ -16,6 +17,7 @@ router.get('/', (req, res) => {
 router.post('/signup', userAuth.validateUser, signup)
 router.post('/login', login )
 router.post('/question', auth_token, postQuestion.createQuestion)
+router.delete('/question', auth_token ,deleteQuestion.deleteQuestion)
 //router.post('/users', createUserController);
```

Comment:
You have not passed the parameter in this route.  Check ln:5 in `controllers/deleteQuestion.js`, you are making a reference to `req.params.id` there so you have to pass the param in the route else `req.params.id` will always be null.

```javascript
router.delete('/question/:id', auth_token ,deleteQuestion.deleteQuestion)
```